### ISSUE TITLE
"undefined(window.navigator.userAgent)" in metamask-onboarding.bundle.js

### ISSUE THREAD

**Issue by cyrusadkisson** [MetaMask/metamask-onboarding#58]
Hi. I'm trying to use metamask-onboarding.bundle.js in my web app and getting an error on the page load. 

```
Uncaught TypeError: undefined is not a function
    at Function.Onboarding._detectBrowser (metamask-onboarding.bundle.js:2441)
    at new Onboarding (metamask-onboarding.bundle.js:2309)
    at initialize (exchange.js:246)
```
When I check out the metamask-onboarding.bundle.js file itself (pulled from node_modules/.../dist/ folder) I see the section in question:

```
  Onboarding._detectBrowser = function () {
            var browserInfo = undefined(window.navigator.userAgent);
            if (browserInfo.browser.name === 'Firefox') {
                return 'FIREFOX';
            }
            else if (['Chrome', 'Chromium'].includes(browserInfo.browser.name || '')) {
                return 'CHROME';
            }
            return null;
        };
```
I'm no javascript god, but that "undefined(...)" looks incorrect to me. Am I right about that? How should I fix?

**Comment by ignaval** [comment]
@cyrusadkisson Change that line to

`var browserInfo = Bowser.parse(window.navigator.userAgent);`

and it should work.

**Comment by Gudahtt** [comment]
Thanks for the report! I think this is caused by the same root problem behind #47. We're working on fixing that now.

**Comment by Gudahtt** [comment]
This should be fixed by #63

**Comment by Gudahtt** [comment]
The fix for this has just been released as v1.0.1
